spec/fixtures/1889/hs_lhglbbiz_17502.json

```diff
@@ -394,6 +395,21 @@
           ],
           "percent": 10
         },
+        {
+          "type": "program_disable",
+          "entity": 671,
+          "entity_type": "player",
+          "created_at": 1615311279,
+          "reason": "1 share(s) bought in IR, end condition met"
+        },
+        {
+          "type": "program_share_pass",
+          "entity": 671,
+          "entity_type": "player",
+          "indefinite": false,
```

Comment:
Question: would it be better to not output fields that have the default value?

Reply:
sure no need

Reply:
Done